### PR TITLE
spelt out what rbac means

### PR DESCRIPTION
**Pull request by jonico**
figured that not everybody knows what rbac stands for

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@ Contents:
     traces
     cli
     authentication
-    rbac
+    rbac (role based access control)
```
Comment (manasdk) [inline docs/source/index.rst]: Will check if it renders properly in the left nav.

Comment (Kami) [inline docs/source/index.rst]: Since it's a link you need to use `title <link syntax>`

**Comment by Kami** [comment]
Thanks, but it actually already renders correctly on the website - http://docs.stackstorm.com/latest/ (see navigation bar on the left).

In this case "rbac" is a link to a file and the title (Role Based Access Control) is read from the referenced / link file (docs/source/rbac.rst).

**Comment by manasdk** [comment]
I think we are ok here as Kami pointed out i.e. RBAC is already spelled out.

**Pull request closed without merging**